docs/src/app/new/(content)/components/collapsible/demos/hero/tailwind/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Collapsible } from '@base-ui-components/react/collapsible';
+
+export default function ExampleCollapsible() {
+  return (
+    <Collapsible.Root
+      className="flex min-h-36 w-56 flex-col justify-center"
+      render={<div />}
+    >
```

Comment:
@mj12albert Collapsible Root accepts DOM props but doesn't render anything by default. Feels like an oversight? I expected it'd render a div like Accordion

Reply:
It was because the ARIA pattern is just a trigger and the panel, though I guess for practical purposes you'd usually want an element, and maybe for consistency too
We can have it render a div but also allow `render={null}` instead

Reply:
Let's just render a DOM element by default. It feels unexpected in comparison to other components that have Root and don't have portalled children

> allow render={null} instead

We don't allow this for other components, no? Don't see a need for it to be fair